src/encoded/static/components/search.js

```diff
@@ -552,52 +554,128 @@ const DatasetComponent = (props, reactContext) => {
     let biosampleTerm;
     let organism;
     let lifeSpec;
+    let assays = [];
```

Comment:
My spider tingles go off when I see an [] and "let" used together. Maybe that is just me remembering the many battles ESLint I had with ESLint and lost badly